data/json/items/tool/misc.json

```diff
@@ -58,6 +83,70 @@
     "color": "brown",
     "use_action": "BELL"
   },
+  {
+    "type": "TOOL",
+    "id": "eac_dab_pen",
+    "name": "dab pen",
+    "name_plural": "dab pens",
+    "description": "A battery operated dab pen used for smoking cannabis distillate cartridges. ",
+    "symbol": "!",
+    "color": "dark_gray",
+    "looks_like": "ecig",
+    "weight": "20 g",
+    "volume": "100 ml",
+    "price": 2500,
+    "material": "plastic",
+    "ammo": "battery",
+    "magazines": [
+      [
+        "battery",
+        [ "light_minus_battery_cell", "light_minus_disposable_cell", "light_battery_cell", "light_disposable_cell" ]
+      ]
+    ],
+    "charges_per_use": 1,
+    "max_charges": 100,
+    "use_action": {
+      "target": "eac_dab_pen_on",
+      "msg": "You turn on the dab pen.",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "The dab pen's batteries need more charge. ",
+      "type": "transform"
+    }
+  },
+  {
+    "type": "TOOL",
+    "id": "eac_dab_pen_on",
+    "name": "dab pen (on)",
+    "name_plural": "dab pens (on)",
+    "description": "dab pen (on), now all you need is a distillate cartridge and you're ready to rock and roll. ",
+    "symbol": "!",
+    "color": "dark_gray",
+    "looks_like": "ecig",
+    "weight": "20 g",
+    "volume": "100 ml",
+    "price": 2500,
+    "material": "plastic",
+    "ammo": "battery",
+    "power_draw": 9000,
+    "revert_to": "eac_dab_pen",
```

Comment:
```suggestion
    "revert_to": "dab pen",
```

Reply:
is revert_to an id? then it should be dab_pen

Reply:
Woops good catch